docs/content/posts/2023-10-10-release-1.18.0.md

```diff
@@ -0,0 +1,572 @@
+automate and simplify common maintenance tasks into small sets of procedures,
+which greatly reduces users' administrative burden.
+
+This new update primarily targets developers of Flink connectors, who can now
+predefine custom stored procedures into connectors via the Catalog interface.
+The primary benefit to users is that connector-specific tasks that previously
+may have required writing custom Flink code can now be replaced with simple
+calls that encapsulate, standardize, and potentially optimize the underlying
+operations. Users can execute procedures using the familiar `CALL` syntax, and
+discover a connector's available procedures with `SHOW PROCEDURES`. Stored
+procedures within connectors improves the extensibility of Flink's SQL and
+Table APIs, and should unlock smoother data access and management for users.
+
+**More Information**
+* [Documentation](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/procedures/)
+* [FLIP-311: Support Call Stored Procedure](https://cwiki.apache.org/confluence/display/FLINK/FLIP-311%3A+Support+Call+Stored+Procedure)
+
+### Extended DDL Support
+
+From this release onwards, Flink supports
+
+- `REPLACE TABLE AS SELECT`
+- `CREATE OR REPLACE TABLE AS SELECT`
+
+and both these commands and previously supported `CREATE TABLE AS` can now support atomicity provided the underlying 
+connector also supports this.
+
+Moreover, Apache Flink now supports TRUNCATE TABLE in batch execution mode. Same as before, the underlying connector needs 
+to implement and provide this capability
+
+And, finally, we have also implemented support for adding, dropping and listing partitions via
+
+- `ALTER TABLE ADD PARTITION`
+- `ALTER TABLE DROP PARTITION`
+- `SHOW PARTITIONS`
+
+**More Information**
+- [Documentation on TRUNCATE](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/sql/truncate/)
+- [Documentation on CREATE OR REPLACE](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/sql/create/#create-or-replace-table)
+- [Documentation on ALTER TABLE](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/sql/alter/#alter-table)
+- [FLIP-302: Support TRUNCATE TABLE statement in batch mode](https://cwiki.apache.org/confluence/display/FLINK/FLIP-302%3A+Support+TRUNCATE+TABLE+statement+in+batch+mode)
+- [FLIP-303: Support REPLACE TABLE AS SELECT statement](https://cwiki.apache.org/confluence/display/FLINK/FLIP-303%3A+Support+REPLACE+TABLE+AS+SELECT+statement)
+- [FLIP-305: Support atomic for CREATE TABLE AS SELECT(CTAS) statement](https://cwiki.apache.org/confluence/display/FLINK/FLIP-305%3A+Support+atomic+for+CREATE+TABLE+AS+SELECT%28CTAS%29+statement)
+
+### Time Traveling
+
+Flink supports the time travel SQL syntax for querying historical versions of data that allows users to specify a point 
+in time and retrieve the data and schema of a table as it appeared at that time. With time travel, users can easily 
+analyze and compare historical versions of data.
```

Comment:
Is it possible to add an example in here ? Like some thing:
"
For example, a user can query a table at a specified point in time with the following statement;
```sql
-- Query the table `tb` for data on November 11, 2022
SELECT * FROM tb FOR SYSTEM_TIME AS OF TIMESTAMP '2022-11-11 00:00:00';
```